Sources/CardPayments/APIRequests/ConfirmPaymentSourceRequest.swift

```diff
@@ -21,16 +21,15 @@ struct ConfirmPaymentSourceRequest: APIRequest {
     ) throws {
         self.jsonEncoder = encoder
         var confirmPaymentSource = ConfirmPaymentSource()
-        var card = cardRequest.card
-        let verification = Verification(method: cardRequest.sca.rawValue)
-        card.attributes = Attributes(verification: verification)
-            
+        
+        let serviceCard = ServiceCard(cardRequest: cardRequest)
+        
         confirmPaymentSource.applicationContext = ApplicationContext(
             returnUrl: PayPalCoreConstants.callbackURLScheme + "://card/success",
             cancelUrl: PayPalCoreConstants.callbackURLScheme + "://card/cancel"
         )
-        
-        confirmPaymentSource.paymentSource = PaymentSource(card: card)
+
+        confirmPaymentSource.paymentSource = PaymentSource(card: serviceCard)
```

Comment:
This `PaymentSource` is currently [public](https://github.com/paypal/iOS-SDK/blob/b0e61d6f27d6d094354f1322f8df2310f15c4ad9/Sources/CardPayments/Models/PaymentSource.swift#L6). I don't think it needs to be.

Reply:
Yes, it's used to format the request body.
```
{
  "application_context": {
    "return_url": "sdk.ios.paypal://card/success",
    "cancel_url": "sdk.ios.paypal://card/cancel"
  },
  "payment_source": {
    "card": {
      "number": "4344511979394247",
      "security_code": "123",
      "billing_address": null,
      "name": null,
      "attributes": {
        "vault": {
          "store_in_vault": "ON_SUCCESS"
        },
        "verification": {
          "method": "SCA_ALWAYS"
        },
        "customer": {
          "id": "test567"
        }
      },
      "expiry": "2025-01"
    }
  }
}

```

Reply:
I see what you mean.